### DISCUSSION TITLE
How do I make a bot/ai move from somewhere other than the debug panel?

### DISCUSSION THREAD

**Discussion by SamuelBucher** [boardgameio/boardgame.io#1155] [Q&A]
Hello. I'm currently working on my own project based on the tutorial tic-tac-toe game, and I need to implement a bot/ai opponent. I have been reading documentation, issues, and discussions on this topic, but I still can't figure out what code creates a bot instance and makes in make a move. Would someone provide an explanation or a very simple example? Thank you in advance.

**Comment by Toshinaki** [comment]
Same question here.
Did you find any solutions to this?
